lib/Service/FsEventService.php

```diff
@@ -36,16 +39,13 @@ public function onAccessUpdateDecl(Node $node, bool $recurse = true): void {
 
 			$files = $this->storageService->getAllFilesInFolder($node);
 		} else {
-			$files = [$node];
+			$files = [$node->getId()];
```

Comment:
```suggestion
			$fileIds = [$node->getId()];
```
would be better to name it like this so there is no confusion. In the other places too.